ansible/plugins/connection/sonic_ssh.py

```diff
@@ -0,0 +1,69 @@
+import imp
```

Comment:
better to rename the file as multi_ssh or multi_passwd_ssh instead of sonic_ssh?

Reply:
Sure.